CONTRIBUTING.md

```diff
@@ -1,35 +1,211 @@
+  * [Must read](#must-read)
+  * [Good to know](#good-to-know)
+
+
+# Submitting an issue/bug report
+
+This section guides you through submitting an issue/bug report for qBittorrent.
+
+Following these guidelines helps maintainers and the community understand your report, reproduce the behavior, and find related reports.
+
+Make sure to follow these rules carefully when submitting a bug report. Failure to do so will result in the issue being closed.
+
+## Bug reporting etiquette
+
+*   Issues and comments on issues must always be in **English**.
+
+*   This project is supported by volunteers, do not expect "customer support"-style interaction.
+
+*   **Be patient**. The development team is small and resource limited. Developers and contributors take from their free time to analyze the problem and fix the issue. :clock3:
+
+*   Harsh words or threats won't help your situation. What's worse, your complain will (very likely) be **ignored**. :fearful:
+
+## What is and actual bug report?
+
+Developers and contributors are not supposed to deal with issues for which little to no investigation to find the actual cause of a purported issue was made by the reporter.
+
+Positive contributions are those which are reported with efforts to find the actual cause of an issue, or at the very least efforts were made to narrow it as much as possible.
+
+Requiring people to investigate as much as possible before opening an issue will more than likely avoid burdening the project with invalid issues or issues unrelated to qBittorrent.
+
+The following are _not_ bug reports; **Check the wiki, forum or other places for help and support for issues like these**:
+
+-   Explanation of qBittorrent options (see wiki);
+-   Help with Web UI setup;
+-   Help with embedded tracker setup;
+-   Help about Bittorrent in general;
+-   Issues with specific search plugins;
+-   Asking for specific builds of qBittorrent not currently officially available;
+    -   If you really need one such older unavailable build, compile it yourself from source (see wiki) or get someone to do it for you on the forum, for example.
+-   Possibly others; read on and use common sense.
+
+
+
+The issue tracker is for provable issues only: You will have to make the case that the issue is really with qBittorrent and not something else on your side.
+
+To make a case means to provide detailed steps so that anybody can reproduce the issue.
+Be sure to rule out that the issue is not caused by something specific on your side.
+
+Any issue opened without effort to provide the required details for developers, contributors or anybody else to reproduce the problem will be closed as invalid.
+For example:
+-   Crash reports with just a stack trace;
+-   Speculated performance issues that do not come with actual profiling data + analysis supporting the claim;
+
+## Before submitting a bug report
+
+-   **Do some basic troubleshooting**:
+    -   restart qBittorrent;
```

Comment:
Capitalize?

Reply:
Pardon, I meant capitalize the first word:
* Restart qBittorrent
* Restart your PC

Reply:
Ok